training-react/src/app/components/LoginForm/index.js

```diff
@@ -5,22 +5,21 @@ import PropTypes from 'prop-types';
 
 import { customInput } from '../Field';
 import userActions from '../../../redux/user/actions';
+import { login } from '../../../helpers/auth';
 
 import styles from './styles.scss';
 import { required, minLength, isEmail } from './validation';
 
 class LoginForm extends Component {
   submit = values => {
-    const { getUser, history } = this.props;
-    getUser(values);
-    history.push('/game');
+    login(values, this.props);
```

Comment:
I don't think we need to extract this to a helper function as dealing with prop should be  in the component scope